string outcome
float64[] forces

